diff --color -ru rt-5.0.4_vanilla/share/html/Elements/Footer rt-5.0.4_front-page-stripdown/share/html/Elements/Footer
--- rt-5.0.4_vanilla/share/html/Elements/Footer	2023-05-03 21:31:56.000000000 +0000
+++ rt-5.0.4_front-page-stripdown/share/html/Elements/Footer	2023-05-05 07:37:27.241701397 +0000
@@ -58,15 +58,21 @@
 %     if ($m->{'rt_base_time'}) {
         <span id="time"><&|/l&>Time to display</&>: <%Time::HiRes::tv_interval( $m->{'rt_base_time'} )%> -</span>
 %     }
+% if (0) {
       <span><&|/l_unsafe, $RT::VERSION, &>RT Version [_1]</&></span>
+% }
     </p>
   </div>
   <div class="col-<% $cols %>">
+% if (0) {
     <p id="bpscredits" class="text-center"><span><&|/l_unsafe, '2023', '&#187;&#124;&#171;', '<a href="https://bestpractical.com/about">Best Practical Solutions, LLC</a>', &>Copyright 1996-[_1] [_2] [_3].</&></span></p>
+% }
   </div>
 % if (!$Menu) {
   <div class="col-4">
+% if (0) {
     <p id="legal" class="text-md-right text-center"><&|/l_unsafe, '<a href="mailto:sales@bestpractical.com">sales@bestpractical.com</a>' &>For support and sales inquiries, please contact [_1].</&></p>
+% }
   </div>
 % }
 </div>
diff --color -ru rt-5.0.4_vanilla/share/html/Elements/Login rt-5.0.4_front-page-stripdown/share/html/Elements/Login
--- rt-5.0.4_vanilla/share/html/Elements/Login	2023-05-03 21:31:56.000000000 +0000
+++ rt-5.0.4_front-page-stripdown/share/html/Elements/Login	2023-05-05 07:36:18.711720110 +0000
@@ -61,7 +61,7 @@
 % $m->callback( %ARGS, CallbackName => 'BeforeForm' );
 
 <div id="login-box">
-<&| /Widgets/TitleBox, title => loc('Login'), titleright => $RT::VERSION, hideable => 0 &>
+<&| /Widgets/TitleBox, title => loc('Login'), titleright => '', hideable => 0 &>
 
 <& LoginRedirectWarning, %ARGS &>
 
