diff -ru DBIx-SearchBuilder-1.67_vanilla/lib/DBIx/SearchBuilder/Handle.pm DBIx-SearchBuilder-1.67_mariadb-fix/lib/DBIx/SearchBuilder/Handle.pm
--- DBIx-SearchBuilder-1.67_vanilla/lib/DBIx/SearchBuilder/Handle.pm	2016-07-18 19:06:53.000000000 +0000
+++ DBIx-SearchBuilder-1.67_mariadb-fix/lib/DBIx/SearchBuilder/Handle.pm	2020-04-16 18:16:46.984876626 +0000
@@ -309,7 +309,9 @@
     my $dbh = $self->dbh;
     return unless $dbh;
     $self->Rollback(1);
-    return $dbh->disconnect;
+    my $res = $dbh->disconnect;
+	$self->dbh(undef);
+	return $res;
 }
 
 
